mailviews/tests/urls.py

```diff
@@ -1,4 +1,4 @@
-from django.conf.urls.defaults import include, patterns, url
+from django.conf.urls import include, patterns, url
```

Comment:
It should be easy enough to just catch the `ImportError` here and change the import.